fluent-plugin-elasticsearch.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'fluentd', '>= 0.14.22'
   s.add_runtime_dependency 'excon', '>= 0'
   s.add_runtime_dependency 'elasticsearch'
+  s.add_runtime_dependency 'elasticsearch-xpack'
```

Comment:
Oops. We might have to specify elasticsearch-xpack gem version dependencies....?
Or, we have to check `Elasticsearch::VERSION` on `#configure`.

Reply:
It should make elasticsearch-xpack optional to support older version, surely.

Reply:
In .gemspec, what is the simple way to resolve dependency?
If gem is already installed, we can use Gem::Specification.select or something but right after `s.add_runtime_dependency 'elasticsearch' ` context, it is not available.
I'll check it further more.

Reply:
we need the already resolved dependency tree to check `elasticsearch` version.

Reply:
`<Gem::Dependency type=:runtime name="elasticsearch" requirements=">= 0">` is not resolved yet, so `s.satisfies_requirement?` does not work as expected...

Reply:
Sorry for confusing. Current implementation's raising error is enough for me....